scripts/dts/edtlib.py

```diff
@@ -2476,6 +2528,8 @@ def _check_dt(dt):
 def _err(msg):
     raise EDTError(msg)
 
+# Regular expression for non-alphanumeric characters.
```

Comment:
Technically this isn't non-alphanumeric, it's non ["word" characters](https://perldoc.perl.org/perlrecharclass#Word-characters), i.e. whatever doesn't match `\w` in a Perl expression.  In the immediate case that means `_` (which isn't alphanumeric) is not in `_NOT_ALPHANUMERIC`.  In a future case, there's unicode.

All of which is to say: maybe use "(Perl) word character" instead of "alpha-numeric" passim.

Reply:
Using re.ASCII changes the semantics so it's not as general as that.

From the re module documentation (note the ASCII part):

```
\W
    Matches any character which is not a word character. This is the opposite of \w. If the ASCII flag is used this becomes the equivalent of [^a-zA-Z0-9_].
```

I forgot that this includes underscores, so I've renamed to `_NOT_ALPHANUM_OR_UNDERSCORE`.